packages/icons/README.md

```diff
@@ -36,3 +36,54 @@ Each Icon component will render an `<svg />` element with an unbounded size and
   
   <CheckCircleFilled style={{ fill: "green", height: "24px", width: "24px" }} />
 ```
+
+## Icons
+
+Included icons in this package are 
+`ArrowLeft`,
```

Comment:
Is worried about having to maintain this list in several places. Having a single source of truth would be ideal. Could we export a list of icon names from the icon package?

Reply:
Now that we have Storybook in a public place I think I will just make that the official list so we don't have to maintain a list in a README